src/openrct2/Context.cpp

```diff
@@ -280,9 +280,30 @@ namespace OpenRCT2
                 config_save_default();
             }
 
+            // TODO add configuration option to allow multiple instances
+            // if (!gOpenRCT2Headless && !platform_lock_single_instance()) {
+            //  log_fatal("OpenRCT2 is already running.");
+            //  return false;
+            // } //This comment was relocated so it would stay where it was in relation to the following lines of code.
+
+            _objectRepository = CreateObjectRepository(_env);
+            _objectManager = CreateObjectManager(_objectRepository);
+            _trackDesignRepository = CreateTrackDesignRepository(_env);
+            _scenarioRepository = CreateScenarioRepository(_env);
+
+            if (!language_open(gConfigGeneral.language))
+            {
+                log_error("Failed to open configured language...");
+                if (!language_open(LANGUAGE_ENGLISH_UK))
+                {
+                    log_fatal("Failed to open fallback language...");
+                    return false;
+                }
+            }
+
             if (platform_process_is_elevated())
             {
-                std::string elevationWarning = "For security reasons, it is strongly recommended not to run OpenRCT2 with elevated permissions.";
+                std::string elevationWarning = language_get_string(6154);
```

Comment:
Create an entry in `string_ids.h` and use it here. Do not use a hardcoded string ID.